### DISCUSSION TITLE
Fermentology unbook, v.001

### DISCUSSION THREAD

**Discussion by micahvandegrift** [knowledgefutures/pubpub#1419] [Show and tell]
Pubbers, 

We've been building a publishy thing in public, while learning the PubPub ropes, and are ready for some feedback. I won't give too much context so the comments can be fresh from your first view, but I'll say that it has felt like we're breaking several publishing molds/mores all at once. 

Can we call this a peer review of the form, type, and execution and also welcome comments on the content? 

[Fermentology](https://fermentology.pubpub.org/) is 'open' for business.

**Comment by jeffpooley** [comment]
This is fantastic! Such a creative use of PubPub’s layout features, and I love the “unbook” concept. Congratulations!

**Comment by gabestein** [comment]
Whoops. I've been waiting to find time to properly comment on this so long that I forgot to comment on it. Long story short: this is super cool, Micah! I love the design, the use of Connections, all the multimedia (and use of collections for multiple entrypoints/exploratory paths), and as a sourdough enthusiast, the content. I've shared it with a ton of friends already, and we'll make sure to highlight in the next newsletter and on social.

This might be a silly question, but I'm really curious to know what you're calling it, internally and externally. As you said, it's breaking quite a few molds here, so...what is it? Not a book. Not the series itself. Way more than a supplement...selfishly, I'd love to find a noun or two for this kind of resource so we can speak to it more effectively when people ask about potential outside-the-box uses.

**Reply by micahvandegrift** [reply]
Yeah, that's the question of the hour @gabestein. When I talk about it to others outside our project team, I refer to it interchangeably as: Hub, Community, Site, Collection, Platform. Internally, we call it a Bucket - all pubs go in the Ferm bucket and then we can creatively offer paths, connections, experiences. But that's just my holdover term from working in repository land for too long. 

You'll notice there is a "book" in there somewhere, so we have defaulted to calling that grouping of items a "Book", but I don't like to and try not to. There is potential for this thing to be a Pre-Book (in conversation with a Press), so this middle thing between the Webinar series and the Printed Book is kinda... a.... frankenbook? ;)
